components/brave_wallet/browser/brave_wallet_provider_impl.cc

```diff
@@ -801,6 +806,7 @@ std::unique_ptr<base::Value> BraveWalletProviderImpl::RequestBaseValue(
 bool BraveWalletProviderImpl::CommonRequestOrSendAsync(
     base::Value input_value,
     const std::string& origin,
+    mojom::CoinType coin,
```

Comment:
Why is there a CoinType param here? This file is only for Ethereum, we can remove this and just pass in Ethereum below.

Reply:
yeah, makes sense. I removed it from `CommonRequestOrSendAsync`